pkgpanda/integration-tests/test_setup.py

```diff
@@ -11,20 +12,42 @@ def tmp_repository(tmpdir, repo_dir="../tests/resources/packages"):
     return repo_path
 
 
+@pytest.fixture(scope="module")
+def preserve_sysctl(request):
+    sys_ctl_param = check_output(["sysctl", "kernel.watchdog_thresh"]).decode("utf-8").strip("\n")
+
+    def fin():
+        if sys_ctl_param is not None:
+            check_call(["sysctl", "-q", "-w", sys_ctl_param])
+
+    request.addfinalizer(fin)
+    return sys_ctl_param
+
+
 def test_setup(tmpdir):
     repo_path = tmp_repository(tmpdir)
     tmpdir.join("root", "bootstrap").write("", ensure=True)
+    sysctl_config = tmpdir.join("sysctl.conf")
```

Comment:
So we're writing the sysctl values out to a temporary file here, which is right, but we're still assuming we can do a `sysctl -w`, which won't work when I (for instance) run this test on my desktop, I'm going to be running it not as root, so won't have permission to do a `sysctl -w`.

Need to either
 -  monkeypatch `activate_sysctl` / https://github.com/dcos/dcos/pull/294/files#diff-f5d4e658e2c0898d8b3d14c39f3c9a8dR53 inside tests, so we test that the call is exactly what we expect, but don't actually do it
 - Make it so the option to use a "rooted" systemctl file also results in activate_sysctl not being run (Something similar happens for the "manage_systemd" option causing us to not manage systemd settings in certain cases, like when running in non-root integration tests).